mParticle-Kochava/MPKitKochava.m

```diff
@@ -230,6 +238,51 @@ - (MPKitExecStatus *)setOptOut:(BOOL)optOut {
     return execStatus;
 }
 
+- (nonnull MPKitExecStatus *)logBaseEvent:(nonnull MPBaseEvent *)event {
+    if ([event isKindOfClass:[MPEvent class]]) {
+        return [self routeEvent:(MPEvent *)event];
+    } else if ([event isKindOfClass:[MPCommerceEvent class]]) {
+        return [self routeCommerceEvent:(MPCommerceEvent *)event];
+    } else {
+        return [[MPKitExecStatus alloc] initWithSDKCode:@(MPKitInstanceKochava) returnCode:MPKitReturnCodeUnavailable];
+    }
+}
+
+- (MPKitExecStatus *)routeEvent:(MPEvent *)event {
+    KVAEvent *kochavaEvent = [KVAEvent eventWithType:KVAEventType.custom];
+    kochavaEvent.customEventNameString = NSStringFromEventType(event.type);
+    kochavaEvent.nameString = event.name;
+    kochavaEvent.infoDictionary = event.customAttributes;
+    [kochavaEvent send];
+    
+    return [[MPKitExecStatus alloc] initWithSDKCode:@(MPKitInstanceKochava) returnCode:MPKitReturnCodeSuccess forwardCount:0];
+}
+
+- (MPKitExecStatus *)routeCommerceEvent:(MPCommerceEvent *)commerceEvent {
+    KVAEvent *kochavaEvent = [KVAEvent eventWithType:KVAEventType.custom];
+    kochavaEvent.customEventNameString = NSStringFromEventType(commerceEvent.type);
+    NSMutableDictionary *info = [commerceEvent.customAttributes mutableCopy];
```

Comment:
I think you need to create a dictionary here if no custom attributes were set to ensure RevenueAmount will be set